client/app/projects/blog/blog.controller.js

```diff
@@ -143,7 +145,9 @@ angular.module('observatory3App')
 
       });
     };
-
+    $scope.readOnly = function(){
```

Comment:
Please use consistent naming.
Below this is called canEdit(), unless there's a specific reason to call this readOnly() it would be ideal to use the same name.